packages/react-components/react-spinbutton/src/stories/SpinButton/SpinButtonAccessibility.md

```diff
@@ -0,0 +1,90 @@
+## Component features and behavior
+
+This section documents accessibility-related behavoirs of spin buttons.
+
+### Keyboarding
+
+#### Tab Order
+1. Value field
+
+#### States
+1. Committed (Focused and value committed)
+2. Editing (Focused and editing)
+
+#### Keyboard State Diagram
+
+| Starting state | Transition | Resulting state |
+| ---------- | ------------ | ------- |
+| Content before spin button | Tab | Committed |
+| Committed  | Tab | Content after spin button (Value committed) |
+| Editing  | Tab | Content after spin button (Value committed) |
+| Editing (Autofill option provided)  | Tab | Content after spin button (Value committed) |
+| Editing  | Enter | Committed |
+| Committed  | Any Edit Key (that results in a change) | Editing |
+| Editing  | Any Edit Key | Editing |
+| Content after spin button (Value committed) | Shift + Tab | Committed |
+
+##### Edit keys
+| Edit Key | Result |
+| ---------- | ------------ |
+| Home | First item in defined range |
+| End | Last item in defined range |
+| Up arrow | One increment higher |
+| Down arrow | One increment lower |
+| Page up | Several increments higher |
+| Page down | Several increments lower |
+| Typing valid value | Valid value |
+| Begin typing valid value | Autofill option provided |
```

Comment:
It's in the [Figma Spec](https://www.figma.com/proto/4ErMjZ0Mi72legBtMAbbEp/SpinButton?node-id=309%3A57799). We could try to discourage non-numeric numbers by removing that section.

Reply:
We intentionally omitted auto complete functionality for now because, as Sarah noted, it's not clearly a valuable feature and  it's worth considering a more general-purpose auto complete implementation that can be used across different types of inputs.